src/modules/hooks/useBlockExplorer/useBlockExplorer.test.ts

```diff
@@ -0,0 +1,28 @@
+import { renderHook } from '@testing-library/react';
+import * as wagmi from 'wagmi';
+import { mainnet, polygon } from 'wagmi/chains';
+import { useBlockExplorer } from './useBlockExplorer';
+
+describe('useBlockExplorer', () => {
+    it('generates correct URL for different entity types and chain IDs', () => {
+        jest.spyOn(wagmi, 'useChains').mockReturnValue([mainnet, polygon]);
```

Comment:
You can create the spy once and reuse it across the tests:
```
const useChainsSpy = jest.spyOn(wagmi, 'useChains');

afterEach(() => {
  useChainsSpy.mockReset();
});
```

Reply:
✅ https://github.com/aragon/ods/pull/220/commits/d917b9ea40962ded12c811ec0c207f7e5ca65be8